04/dbernhard/tests_if7.oat

```diff
@@ -0,0 +1,15 @@
+int program(int argc, string[] args) {
+    return function(0) + function(1) + function(2) + function(3);
+}
+
+int function(int x) {
+    if (x == 0) {
```

Comment:
I am not sure that this should be accepted as valid oat.

Reply:
For the block in the if_stmt rule we choose the empty list (same as args can be empty in a fdecl)

Reply:
I meant the fact that we have an inplicit return 0 statement

Reply:
Now I understand what you meant. I will remove this test.